modules/sdl2/sdl.c

```diff
@@ -30,6 +30,25 @@ struct vidisp_st {
 static struct vidisp *vid;
 
 
+static enum vidfmt match_fmt(u_int32_t fmt)
```

Comment:
the input parameter type should be: `enum vidfmt`

the return type should be `Uint32` (same as input to `SDL_CreateTexture`).